### PR TITLE
Update French (Canada) and Japanese translations

### PR DESCRIPTION
**Pull request by OmarIthawi**
I've done the following:

 - Marked all the French Canadian [hawthorn](https://www.transifex.com/open-edx/open-edx-releases/translate/#fr_CA/release-hawthorn/)/[-js](https://www.transifex.com/open-edx/open-edx-releases/translate/#fr_CA/release-hawthorn/) as reviewed.
 - Same for Japanese ([more info on the Open edX Slack](https://openedx.slack.com/archives/C1KHYD4LT/p1565103280006200?thread_ts=1562097426.000900&cid=C1KHYD4LT)).
 - Updated Japanese and Canadian French via `$ paver i18n_robot_pull`
 - Committed and pushed to GitHub.

This is mainly to resolve a couple of [PSAC reported issues](https://docs.google.com/document/d/172ao8oPRToeqA_Y7PDCRSl6UM96S7TtfyUiwwZnRfLA/edit).

### CODE REVIEW

File: cms/static/js/i18n/fr-ca/djangojs.js [hunk 1]
```diff
@@ -413,6 +493,14 @@
     "Chapter name is required": "Le nom du chapitre est requis", 
     "Chapter {order}": "Chap\u00eetre {order}", 
     "Check Your Email": "V\u00e9rifiez votre courriel", 
+    "Check the box to remove %(count)s flag.": [
```
Comment (johnbaldwin) [inline cms/static/js/i18n/fr-ca/djangojs.js]: @OmarIthawi singular 'flag'?

File: cms/static/js/i18n/fr-ca/djangojs.js [hunk 2]
```diff
@@ -2231,6 +2355,10 @@
     "Zoom Out": "R\u00e9duire", 
     "[no tags]": "[aucun tag]", 
     "a day": "un jour", 
+    "about %d hour": [
```
Comment (johnbaldwin) [inline cms/static/js/i18n/fr-ca/djangojs.js]: How is plural handled?

**Comment by johnbaldwin** [comment]
@OmarIthawi I skimmed over one of the files and see what look like three cases:

```
"there is currently {numVotes} vote": [
	      "il y a actuellement {numVotes} vote", 
	      "il y a actuellement {numVotes} votes"
	    ], 
```

For the english expression, is that always singular?

**Comment by OmarIthawi** [comment]
Thanks @johnbaldwin.

> @OmarIthawi I skimmed over one of the files and see what look like three cases:
> 
> ```
> "there is currently {numVotes} vote": [
> 	      "il y a actuellement {numVotes} vote", 
> 	      "il y a actuellement {numVotes} votes"
> 	    ], 
> ```
> 
> For the english expression, is that always singular?

So, not really. The file has been generated by a script.

Looking at the corresponding JavaScript-based functions:

https://github.com/appsembler/edx-platform/blob/f1cc0042300ffdd59b288fd180baa4f62bdafaef/cms/static/js/i18n/ar/djangojs.js#L8-L15

https://github.com/appsembler/edx-platform/blob/f1cc0042300ffdd59b288fd180baa4f62bdafaef/cms/static/js/i18n/ar/djangojs.js#L1947-L1954

It looks like, unlike the `po` files which preserves the original English plural form, the JavaScript file ignores it. Mostly a space-saving measure since the English plural form is useless. 

That's what I think, please let me know what do you think?

**Comment by OmarIthawi** [comment]
Thanks @johnbaldwin, it's literally just [the two function above](https://github.com/appsembler/edx-platform/pull/431#issuecomment-518775546) :point_up: 

Yes, there's a lot of more context and tooling around this to get to those functions, but that's what the user needs on their browsers to be able to see Arabic (and other languages) for the JavaScript strings.

**Pull request merged**